### DISCUSSION TITLE
May we create a password protected docx file with this lib?

### DISCUSSION THREAD

**Discussion by john-hu** [dolanmiu/docx#1066] [Q&A]
We can create a docx with this. But, we have no idea how to add a password to the output docx file. May you tell us if there is any way to do that?

**Comment by dolanmiu** [comment]
Not yet im afraid, is there OOXML documentation for this feature?

**Reply by dolanmiu** [reply]
https://stackoverflow.com/questions/858232/office-open-xml-ooxml-specification-encryption

Just did some googling

It seems password protected word documents is not part of the OOXML spec and is vendor specific.
